lib/index.js

```diff
@@ -541,9 +552,8 @@ function _connect(options, callback) {
   if(!options.init) {
     logger.info('connecting to database: ' + _sanitizeUrl(options.url));
   }
-
-  let connectOptions = {...config.connectOptions};
-
+  const {writeConcern} = config.writeOptions;
+  let connectOptions = {...config.connectOptions, writeConcern};
   // convert legacy connect options
   if('socketOptions' in config.connectOptions) {
     connectOptions = {
```

Comment:
[Late to the party] Is it ok that this will overwrite the options set above?

Reply:
yeah @dlongley wants `writeConcern` to overwrite in connectOptions, but not in `gridFsBucket`.